config/initializers/00_random.rb

```diff
@@ -0,0 +1,32 @@
+# -*- coding: us-ascii -*-

```

Comment:
Why is this change of encoding necessary? I'm also not sure if this is a correct way to change it :)

Reply:
Also, can you please add a comment on why this file is necessary? Basically, you can copy-paste the PR description. Thank you!

Reply:
> Why is this change of encoding necessary? I'm also not sure if this is a correct way to change it :)

Another file from config/initializers has been taken as "template" for this file. So this is basically, why.

Reply:
> Also, can you please add a comment on why this file is necessary? Basically, you can copy-paste the PR description. Thank you!

Please check the issue corresponding to this pull request: this file is required to have Random::urandom(size) behave well on Docker for Synology, When running ruby/dawarich on Synology, I am getting the error "failed to get urandom". This file "replaces" the original Random::urandom(size) with a call to OpenSSL's OpenSSL::Random.random_bytes(n), which is apparently working on Docker for Synology.

Reply:
> Another file from config/initializers has been taken as "template" for this file. So this is basically, why.

There is no mention of `# -*- coding: us-ascii -*-` anywhere in the project, so it's safe to remove :)

> Please check the issue corresponding to this pull request: this file is required to have Random::urandom(size) behave well on Docker for Synology, When running ruby/dawarich on Synology, I am getting the error "failed to get urandom". This file "replaces" the original Random::urandom(size) with a call to OpenSSL's OpenSSL::Random.random_bytes(n), which is apparently working on Docker for Synology.

I understand, what I meant is to provide a one-line comment to describe the purpose of the file so it would be easier to understand without having to navigate to GitHub. The code fixes a very specific problem which is usually non-existent in typical Rails project, so it's good to have a small comment to make it clear why it's here.

Reply:
> I understand, what I meant is to provide a one-line comment to describe the purpose of the file so it would be easier to understand without having to navigate to GitHub. The code fixes a very specific problem which is usually non-existent in typical Rails project, so it's good to have a small comment to make it clear why it's here.

Fixes ```failed to get urandom``` for running Ruby on Docker for Synology.

OK? :)

Reply:
Perfect!